ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts

```diff
@@ -92,8 +92,13 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
           this.platform = this.platforms.find(platform => platform.id == this.environmentFormGroup?.controls['platform'].value);
         }
         if (this.platform == null) {
-          this.platform = this.platforms[0];
-          this.environmentFormGroup?.controls['platform']?.setValue(this.platform?.id);
+          if(!this.isHybrid){
+            this.platform = this.platforms[0];
+            this.environmentFormGroup?.controls['platform']?.setValue(this.platform?.id);
+          } else if (this.version.workspace.isWeb) {
+            this.platform = this.platforms.find(platform=>platform.name == this.agent.getPlatformFromOsType(this.agent.osType))
+            this.environmentFormGroup?.controls['platform']?.setValue(this.platform.name);
```

Comment:
platform formControl value either id/value?

Reply:
here both id and name returns same value.